### DISCUSSION TITLE
Treesitter indentation for C is not clang format

### DISCUSSION THREAD

**Discussion by mendax1234** [nvim-treesitter/nvim-treesitter#6519] [Q&A]
I have enabled the indent in my lua config, but it still doesn't use clang format. So I want to confirm whether clang format is used in treesitter for c?

Thanks!

**Comment by clason** [comment]
It is not.

**Reply by mendax1234** [reply]
So, which format is used in treesitter for c? And is there a way for me to specify to use clang format in treesitter for c?
Thanks!

**Reply by clason** [reply]
The Neovim indent settings. And, no, apples and oranges. If you want to use clang format, just `clang-format`.

**Reply by mendax1234** [reply]
Sorry, what do you mean by "just `clang-format`"? So, it is impossible to set tree sitter to clang format? But how can I use clang format?
Thanks!

**Reply by clason** [reply]
Yes, it is impossible. And I can't help you with clang.

**Reply by mendax1234** [reply]
oh, thanks. I see.

**Comment by mendax1234** [comment]
> It is not.



> The Neovim indent settings. And, no, apples and oranges. If you want to use clang format, just `clang-format`.



> Yes, it is impossible. And I can't help you with clang.
